app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabsHeaderViewHolder.kt

```diff
@@ -52,11 +52,10 @@ class RecentTabsHeaderViewHolder(
             HomeSectionHeader(
                 headerText = stringResource(R.string.recent_tabs_header),
                 description = stringResource(R.string.recent_tabs_show_all_content_description_2),
-                onShowAllClick = {
-                    dismissSearchDialogIfDisplayed()
-                    interactor.onRecentTabShowAllClicked()
-                }
-            )
+            ) {
+                dismissSearchDialogIfDisplayed()
+                interactor.onRecentTabShowAllClicked()
+            }
```

Comment:
Let's undo this. I prefer if we were more explicit here.

Reply:
Done.